### ISSUE TITLE
Media browser dialog opens twice for image fields

### ISSUE THREAD

**Issue by stooit** [govCMS/GovCMS7#132]
When using the Media Browser on an image field the overlay opens twice, and you cannot select images until you close the top overlay (an error "You have not selected anything!" is thrown).

This issue has been replicated on production sites and a vanilla local install of govCMS master. It looks like this issue was introduced recently, but I haven't dug into by what.

This is a duplicate of the issue posted on Drupal.org (https://www.drupal.org/node/2534724).

Pull request incoming to incorporate patch 53 from the d.o issue.

**Comment by aleayr** [comment]
Looks like there is a more recent patch from the issue queue, patch 72, might be good to roll that in instead.

**Comment by kurtfoster** [comment]
@srowlands , @aleayr , I'm still getting this issue on beta 9. We're going live very two weeks from now and content entry on our job starts tomorrow. Is this bug fix included in beta10 and if so, when can we expect a release?

**Comment by stooit** [comment]
@kurtsalsa I resubmitted the PR for this a few days ago, hopefully it gets reviewed and merged soon. Has been bugging me for months!

**Comment by kurtfoster** [comment]
@srowlands, ok, thanks for the info. For now I've manually added the patch to my repo so that content entry can move ahead properly this week.

**Comment by Feng-Shui** [comment]
@srowlands Thanks for sorting this, applied the patch this morning and it's working nicely for us.  We're going to patch locally too until this gets merged in, been a real frustration for out editors.

**Comment by andmair** [comment]
@fiasco @gollyg this issue is also a problem for NFSA, which is a very media rich site.  They will need this patch on ACFS before they go live in October.

**Comment by fiasco** [comment]
#227 is the current pending PR which requires a re-roll. @andmair if we can get that re-rolled we'll be in a position technically to commit it (assuming it passes testing and gets in-roadmap).

**Comment by stooit** [comment]
@fiasco re-rolled, you should be able to confirm the fix in the PR environment. It should be noted this patch has had fairly thorough testing; it's in use on many govCMS projects outside of SaaS without known issue

**Comment by fiasco** [comment]
Thanks @srowlands.

@aleayr could we prioritise @Venkata-modgu to test this?

**Comment by aleayr** [comment]
@Venkata-modgu Can we get this tested please?

**Comment by Venkata-modgu** [comment]
Media browser interface opening only once.
Test passed.
